server/routes.go

```diff
@@ -80,8 +80,9 @@ type RESTServer struct {
 	Fixity        FixityDB
 	DisableFixity bool
 
-	server httpdown.Server // used to close our listening socket
-	txgate *util.Gate      // limits number of concurrent transactions
+	server   httpdown.Server // used to close our listening socket
+	txgate   *util.Gate      // limits number of concurrent transactions
+	TapeUsed bool            // Is Bendo reading/writing from tape?
```

Comment:
OK.

On Thu, Aug 25, 2016 at 12:35 PM, Don Brower <notifications@github.com>
wrote:

> In server/routes.go
> <https://github.com/ndlib/bendo/pull/136#discussion_r76278741>:
>
> > @@ -80,8 +80,9 @@ type RESTServer struct {
> >  	Fixity        FixityDB
> >  	DisableFixity bool
> >
> > -	server httpdown.Server // used to close our listening socket
> > -	txgate *util.Gate      // limits number of concurrent transactions
> > +	server   httpdown.Server // used to close our listening socket
> > +	txgate   *util.Gate      // limits number of concurrent transactions
> > +	TapeUsed bool            // Is Bendo reading/writing from tape?
>
> Make this an unexported structure member.
>
> I also like keeping the name similar to the variable in items. So rename
> to useTape?
>
> This variable technically needs a mutex, since it can be changed once the
> server is running. I am only pointing it out for completeness, and I am not
> asking you to guard it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ndlib/bendo/pull/136/files/116b45889c785d5598950eba80d6730d33f98945#r76278741>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADNGTKft1Teltfed6PSS333KFnzDE5vwks5qjcRcgaJpZM4JsckT>
> .
>